ui/src/router/components/form/components/ensembler_config/RouteSelectionPanel.js

```diff
@@ -57,20 +57,16 @@ export const RouteSelectionPanel = ({
   const routeOptions = routes
     .filter((e) => !!e.id)
     .map((e) => {
-      const isDisabled =
```

Comment:
For the UI flow, I think it makes sense to keep the validation on the fallback route options and disable those options that don't meet the criteria (if traffic rules are present, the route should be added to all traffic rules and the default rule). The reasons are:
1. It helps to give users get early feedback rather than wait for the form validation to trigger failures.
2. The final route (No ensembler) or fallback route (standard ensembler) config is usually done after the routes and traffic rules. In a scenario where the user tries to submit the form with an error, any subsequent changes will keep re-triggering the validation. And so, if they keep selecting bad route options in the ensembler section, the form will keep jumping back to the routes panel with the validation error.

Reply:
>that don't meet the criteria (if traffic rules are present, the route should be added to all traffic rules and the default rule)

I might be misunderstanding, but wouldn't this mean any route can be selected for the fallback route, so why would there be a need to disable the options?

Reply:
A route can be selected as fallback if:
* There are no traffic rules on the router
* There are traffic rules on the router and this route appears in all the rules including the default rule

If there are traffic rules on the router and a route doesn't appear in all rules (including default rule), we can disable it from this dropdown and prevent it from being selected in the first place instead of picking up this error on validation. Does it make sense?

Reply:
Clear to me, I've updated the condition for this!